docs/user-guide/timeouts/timeouts-main.md

```diff
@@ -0,0 +1,331 @@
+
+The exact classifications are made from experience with these statistics.
+ - LOW: unlikely to be a reason for a timeout
+ - MEDIUM: might be a reason for a timeout; the timeout might also be a result
+   of the combined complexity with other measures
+ - HIGH: likely to be a reason for a timeout, even if it is the only aspect of
+   the verification problem that shows high complexity
+
+These categories map to intervals as follows.
+
+|    | LOW | MEDIUM | HIGH |
+|----|-----|--------|------|
+| Path count | 0 to 2<sup>20</sup> | 20<sup>20</sup> to 2<sup>80</sup> | > 2<sup>80</sup> |
+| Nonlinear operations | 0 to 10 | 10 to 30 | > 30 |
+
+```{note}
+All heuristics mentioned here are might be improved over time, so 
+consider them subject to change.
+```
+
+(timeout_tac_reports)=
+### Timeout TAC Reports
+
+For each verification item, there is a TAC graph linked in the verification
+report. In case of a timeout this graph contains information one which parts of
+the program were part of the actual timeout, and which were already solved
+successfully. It also contains statistics on the above-described timeout causes.
+
+Find more documentation on TAC reports in general [here](tac-reports).
+
+In the timeout case, the TAC reports contain some additional information that
+should help with diagnosing the timeout.
+
+(timeout_prevention)=
+# Timeout Prevention
+
+Timeout prevention approaches fall into these categories.
+1. Changing tool settings
+2. Changing specs
+3. Changing source code
+
+Checking tool settings is least invasive and easy to do, so it is preferred.
+However, there are cases when parts of the input code that are very tricky need
+to be worked around. Sometimes a combination of approaches is needed to resolve
+a timeout.
+
+
+In the following we will discuss some concrete approaches to timeout prevention.
+This collection will be extended over time based on user's experiences and tool
+improvements.
+
+```{note}
+The old documentation has a section on
+{doc}`troubleshooting </docs/confluence/perplexed>` that addresses timeouts, 
+which might complement the information given here.  
+There is also some helpful information in the section on
+{ref}`summarization <old-summary-example>`.
+Some of the information in these references is out of date.
+```
+
+## Prover Settings
+
+In this subsection we list some option combinations that have helped preventing
+timeouts in the past. We group the options by timeout causes they are most
+relevant for. 
+
+### Dealing With a High Path Count
+
+The Certora Prover internally divides each verification condition into smaller
+subproblems and attempts to solve them separately. This technique is called
+*control flow splitting*. For a more detailed explanation of how control flow
+splitting works, see [this page](control-flow-splitting).
+
+We list a few option combinations that can help in various settings. There is
+a tradeoff between spending time in different places: The Prover can either try
```

Comment:
Sometimes you write `The Prover` and sometimes `The Certora Prover`

Reply:
yeah .. like the long form first, and shorter version later is the plan ... (like people use pronouns, but more explicit) -- you think it's bad style?..